pom.xml

```diff
@@ -195,12 +195,12 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
 
     <!-- Maven protoc compiler -->
     <protobuf-maven-plugin.version>0.5.1</protobuf-maven-plugin.version>
-    <grpc.protobuf-compile.version>3.12.0</grpc.protobuf-compile.version>
+    <grpc.protobuf-compile.version>3.19.2</grpc.protobuf-compile.version>
     <os-maven-plugin.version>1.5.0.Final</os-maven-plugin.version>
 
-    <netty.version>4.1.63.Final</netty.version>
-    <io.grpc.version>1.38.0</io.grpc.version>
-    <tcnative.version>2.0.38.Final</tcnative.version> <!-- See table for correct version -->
+    <netty.version>4.1.74.Final</netty.version>
+    <io.grpc.version>1.44.0</io.grpc.version>
+    <tcnative.version>2.0.48.Final</tcnative.version> <!-- See table for correct version -->
```

Comment:
Thanks @adoroszlai for this upgrade to the build.  We discussed previously that by upgrading `Netty` to _4.1.74_, the bill-of-materials, BOM, contains the versioning dependencies for netty and tcnative (not so for the previous version).  Since the versioning of netty & tcnative is controlled by the BOM, can we _**remove**_ the explicit `tcnative.version` declaration here and its use in the supporting pom.xml files?  
The versioning between netty and tcnative should be set by the BOM now if I understood you correctly. I removed the explicit version property for tcnative from the main pom file and its uses for `netty-tcnative` in the ozone-manager and ozone-common, however I needed to set the classifier.  Does removing the explicit versioning property for `tcnative.version` and relying on the BOM to maintain consistency work for our build environment? 

Previous comment on netty bom upgrade: https://github.com/apache/ozone/pull/2945#discussion_r811622506

Reply:
Thanks @neils-dev for the reminder.  I plan to do this separately.